pkg/demoinfocs/common/player.go

```diff
@@ -447,6 +447,14 @@ func (p *Player) MoneySpentThisRound() int {
 	return getInt(p.resourceEntity(), "m_iCashSpentThisRound."+p.entityIDStr())
 }
 
+// LastPlaceName returns the string value of the player's position.
+func (p *Player) LastPlaceName() string {
+	// if the property doesn't exist we return empty string by default
+	val, _ := p.resourceEntity().PropertyValue("m_szLastPlaceName."+p.entityIDStr())
+
+	return val.StringVal
```

Comment:
this prop is actually not on the resource-entity but on the player entity ^^

```suggestion
	getString(p.Entity, "m_szLastPlaceName")
```